app/utils/json_handler.py

```
@@ -0,0 +1,154 @@
+"""
+JSON 파일 읽기/쓰기 유틸리티
+- filelock을 이용한 크로스 플랫폼 동시성 제어
+"""
+import json
+import os
+from typing import Any, Callable
```

Ruff (0.14.11)

Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)